hexa/workspaces/models.py

```diff
@@ -82,12 +83,49 @@ def delete_if_has_perm(self, *, principal: User):
         self.delete()
 
 
+class WorkspaceMembershipQuerySet(BaseQuerySet):
+    def filter_for_user(
+        self, user: typing.Union[AnonymousUser, User]
+    ) -> models.QuerySet:
+
+        return (
+            self.all()
+            if WorkspaceMembership.objects.filter(
```

Comment:
I think you need to add a `.exists()` or `.first()` at the end to this no ? Here I think you just check if it returns a query set

Reply:
Yes exactly. I'll add it